charts/redpanda/values.yaml

```diff
@@ -41,6 +41,10 @@ nodeSelector: {}
 # For details,
 # see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/).
 tolerations: []
+# -- Allows to specify affinity for Jobs.
```

Comment:
Could you move this and place one in each of the following fields

```
post_upgrade_job:
```
and
```
post_install_job:
```
So they can take affinity rules to target those jobs only. The statefulsets have their own pod affinities also. We can later do something like this and then make the individual ones as overrides, i think we have used that pattern in multiple places at this point.

Reply:
I moved it to each field in this MR. Yes, I want to override it globally but it would be also convenient for fine-tuning.